dm/relay/local_reader.go

```diff
@@ -74,6 +73,8 @@ type BinlogReader struct {
 	// ch with size = 1, we only need to be notified whether binlog file of relay changed, not how many times
 	notifyCh chan interface{}
 	relay    Process
+
+	currentUUID string
```

Comment:
please add some commnet to explain this var, for some new develpor who are not familiar with mysql relay logic